modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+Management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted.
+
+Hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+Hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+Hosted control plane infrastructure:: Resources on the management cluster or external cloud provider that are prerequisites to run hosted control plane processes.
+
+Management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+Hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+[id="hosted-control-planes-personas_{context}"]
+== Personas
+
+Cluster service provider:: Users who assume this role are usually cluster admins on the management cluster and have RBAC to monitor and own the availability of the HyperShift Operator as well as the control planes for the tenant's hosted clusters. The cluster service provider persona is responsible for the several activities, including, but not limited to, the following examples:
+** Owning service-level objects for control plane availability, uptime, and stability.
+** Configuring the cloud account for the management cluster to host control planes
+** Configuring the user-provisioned infrastructure, which includes the host awareness of available compute
+
+Cluster service consumer:: Users who assume this role can request control planes and workers, drive updates, or modify externalized configurations. Typically, this user does not manage or access cloud credentials or infrastructure encryption keys. The cluster service consumer persona can request hosted clusters and interact with node pools. Users who assume this role have Role-Based Access Control to create, read, update, or delete hosted clusters and node pools within a logical boundary.
+
+Cluster instance administrator:: The role that is assumed by the user who is the equivalent to the administrator in stand-alone {product-title}. This user has the `cluster-admin` role in the provisioned cluster, but might not have power over when or how the cluster is updated or configured. This user might have read-only access to see some configuration projected into the cluster.
+
+Cluster (instance) user:: The role that is assumed by the user who is the equivalent of a developer in stand-alone {product-title}. This user does not have a view into OperatorHub or machines.
```

Comment:
Do we use (instance) or just "instance" consistently? I.e., Cluster instance administrator vs. Cluster (instance) administrator.

Reply:
Good catch! I copied that verbatim from the upstream docs, but you're right that we should be consistent. I did a few searches in the OpenShift docs as well as on Google, and I didn't find any hits for the specific phrases "cluster instance user" or "cluster instance administrator". That makes me think that we can probably omit the "instance" and just say "Cluster administrator" and "Cluster user". Do you know of any reason why we'd need to keep the "instance"?

Reply:
Agree :)

Reply:
Thanks for the clarification! In that case, I'll leave "instance" in, without the parentheses for consistency.